lib/kamal/commands/proxy.rb

```diff
@@ -16,8 +16,16 @@ def start
     docker :container, :start, container_name
   end
 
-  def stop(name: container_name)
-    docker :container, :stop, name
+  def stop
+    docker :container, :stop, container_name
+  end
+
+  def pause_app
+    docker :exec, container_name, "kamal-proxy", "pause", "#{config.service}-web"
```

Comment:
This should probably use the `primary_role_name`, there might be a helper method already too that's service + primary role.

Reply:
@nickhammond is this what you intended?

https://github.com/basecamp/kamal/pull/1047/commits/693328cf5bc9ea3209a4920d0c78f3e51537ee46